Userland/Libraries/LibC/string.cpp

```diff
@@ -151,6 +151,19 @@ void* memcpy(void* dest_ptr, void const* src_ptr, size_t n)
 #endif
 }
 
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/memccpy.html
+void* memccpy(void* dest_ptr, void const* src_ptr, int c, size_t n)
+{
+    u8* pd = (u8*)dest_ptr;
+    u8 const* ps = (u8 const*)src_ptr;
```

Comment:
I'm aware that the other functions here don't necessarily do this either, but please try to avoid introducing any new C-style casts.

Reply:
Fixed the casts here but didn't touch the other functions.